pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -161,3 +190,37 @@ func (e *Executor) getMentionedAccounts(event model.NotificationEventType) ([]st
 
 	return notification.FindSlackAccounts(event), nil
 }
+
+func (e *Executor) validateApproverNum(ctx context.Context, approver string, num int) (string, bool) {
+	if num <= 1 {
+		e.LogPersister.Infof("Got approval from \"%s\"", approver)
```

Comment:
nit: https://pkg.go.dev/fmt#hdr-Printing
```suggestion
		e.LogPersister.Infof("Got approval from %q", approver)
```